examples/ffdmd/Si-drive/README

```diff
@@ -0,0 +1,24 @@
+This is an example for a time dependent drive in the smotion class involving a solid.
```

Comment:
An explanation of what is the form of the time-dependent drive would help understand why you need all of those manipulations below

Reply:
They are not needed and have been removed.